edit-post/index.js

```diff
@@ -66,8 +65,12 @@ export function initializeEditor( id, postType, postId, settings, overridePost )
 		'core/editor.publish',
 	] );
 
+	if ( initialEdits ) {
```

Comment:
>My thinking also is that ultimately we should get rid of initializeEditor function entirely, It should be just an Editor component exposed in edit-post

That's an interesting thought! It also solves another issue where it's hard to extract things like  template synchronization out of EditorProvider because it relies on the post entity having been resolved; something we can't do from `initializeEditor`, but we could do from the (`edit-post`) Editor component.